src/command/CommandListBuilder.hxx

```diff
@@ -54,7 +54,7 @@ class CommandListBuilder {
 	/**
 	 * Memory consumed by the list.
 	 */
-	size_t size;
+	size_t size = 0;
```

Comment:
This commit is wrong because `size` is meant to be inialized in`Begin()`. 
Whatever you have been doing, your code probably abused the class in a way it was never intended to be used.

Reply:
Yes. You are right. My mistake.